crates/resolver/src/config.rs

```diff
@@ -337,7 +337,7 @@ pub struct NameServerConfig {
     pub tls_dns_name: Option<String>,
     /// Default is to distrust negative responses from upstream nameservers
     ///
-    /// Currently only SERVFAIL responses are continued on, this may be expanded to include NXDOMAIN or NoError/Empty responses
+    /// Responses of type SERVFAIL, NXDOMAIN and NoError/Empty responses are continued on.
```

Comment:
Actually I don't find the wording here ("continued on") that clear. Maybe that can improved, but I'm not sure how best to describe the effect it has on how the resolver operates.

Reply:
"continues" is in the context of "continues to try other registered nameservers".

Reply:
I've made an attempt to make this clearer in the docs. Let me know what you think?

Reply:
I think this is much clearer. Thank you.